src/libertem/analysis/fem.py

```diff
@@ -0,0 +1,25 @@
+from libertem.viz import visualize_simple
+
+from .base import BaseAnalysis, AnalysisResult, AnalysisResultSet
+
+import libertem.udf.FEM as FEM
+
+
+class ApplyFEM(BaseAnalysis):
```

Comment:
What do you think about keeping in the naming scheme of `SomethingAnalysis` here? That's currently used by all other analyses